azure-pipelines.yml

```diff
@@ -25,12 +22,32 @@ resources:
     name: SkillsFundingAgency/das-platform-building-blocks
     ref: refs/heads/master
     endpoint: GitHub (SFA)
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.4.0
+    endpoint: GitHub (SFA)
+  pipelines:
+  - pipeline: das-employer-config
+    project: Digital Apprenticeship Service
+    source: das-employer-config
+    branch: das-onboarding-initial-config
```

Comment:
is this branch in master yet?

Reply:
I have spoken to Dan and he says when this PR is approved the config will be pushed to Master